### PR TITLE
✨ Fork media amp-ad support

### PR DESCRIPTION
**Pull request by vineeth-forkmedia**
Adds amp-ad support for Fork Media

### CODE REVIEW

File: ads/forkmedia.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function forkmedia(global, data) {
+  validateData(data, ['product']);
```
Comment (calebcordry) [inline ads/forkmedia.js]: did you want to validate `data-html-access-allowed` here?

Comment (vineeth-forkmedia) [reply ads/forkmedia.js]: Tried adding `validateData(data, ['htmlAccessAllowed'])` after Line 27 but validation is failing.

Comment (calebcordry) [reply ads/forkmedia.js]: ok, thanks for checking. Never mind then :)

**Comment by calebcordry** [comment]
Thanks again!

**Comment by vineeth-forkmedia** [comment]
Thank you!

**Pull request merged**